models/hh_psc_alpha_clopath.h

```diff
@@ -0,0 +1,436 @@
+weight 1.0 results in a peak current of 1 pA.
+
+
+(2) Spike Detection
+Spike detection is done by a combined threshold-and-local-maximum search: if
+there is a local maximum above a certain threshold of the membrane potential,
+it is considered a spike.
+
+Parameters:
+
+The following parameters can be set in the status dictionary.
+
+V_m        double - Membrane potential in mV
+E_L        double - Resting membrane potential in mV.
+g_L        double - Leak conductance in nS.
+C_m        double - Capacity of the membrane in pF.
+tau_ex     double - Rise time of the excitatory synaptic alpha function in ms.
+tau_in     double - Rise time of the inhibitory synaptic alpha function in ms.
+E_Na       double - Sodium reversal potential in mV.
+g_Na       double - Sodium peak conductance in nS.
+E_K        double - Potassium reversal potential in mV.
+g_K        double - Potassium peak conductance in nS.
+Act_m      double - Activation variable m
+Act_h      double - Activation variable h
+Inact_n    double - Inactivation variable n
+I_e        double - Constant external input current in pA.
+
+Clopath rule parameters:
+u_bar_plus    double - Low-pass filtered Membrane potential in mV.
+u_bar_minus   double - Low-pass filtered Membrane potential in mV.
+u_bar_bar     double - Low-pass filtered u_bar_minus in mV.
+A_LTD         double - Amplitude of depression in 1/mV.
+A_LTP         double - Amplitude of facilitation in 1/mV^2.
+theta_plus    double - threshold for u in mV.
+theta_minus   double - threshold for u_bar_[plus/minus] in mV.
+A_LTD_const   bool   - Flag that indicates whether A_LTD_ should
+                       be constant (true, default) or multiplied by
+                       u_bar_bar^2 / u_ref_squared (false).
+delay_u_bars  double - Delay with which u_bar_[plus/minus] are processed
+                       to compute the synaptic weights.
+U_ref_squared double - Reference value for u_bar_bar_^2.
+
+Problems/Todo:
+
+better spike detection
+initial wavelet/spike at simulation onset
+
+References:
+
+Spiking Neuron Models:
+Single Neurons, Populations, Plasticity
+Wulfram Gerstner, Werner Kistler,  Cambridge University Press
+
+Theoretical Neuroscience:
+Computational and Mathematical Modeling of Neural Systems
+Peter Dayan, L. F. Abbott, MIT Press (parameters taken from here)
+
+Hodgkin, A. L. and Huxley, A. F.,
+A Quantitative Description of Membrane Current
+and Its Application to Conduction and Excitation in Nerve,
+Journal of Physiology, 117, 500-544 (1952)
+
+Clopath et al., Connectivity reflects coding:
+a model of voltage-based STDP with homeostasis.
+Nature Neuroscience, 13:3, 344-352  (2010)
+
+Clopath and Gerstner, Voltage and spike timing interact
+in STDP – a unified model.
+Front. Synaptic Neurosci., 2:25, (2010)
+doi: 10.3389/fnsyn.2010.00025
+
+Voltage-based STDP synapse (Clopath et al. 2010) connected to a Hodgkin Huxley
+neuronon on ModelDB:
```

Comment:
Typo `neuronon` -> `neuron` and maybe add a `-` between Hodgkin and Huxley for consistency.